### PR TITLE
created user model

### PR DESCRIPTION
**Pull request by randallkanna**

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import DS from 'ember-data';
+
+App.User = DS.Model.extend({
```
Comment (twokul) [inline app/models/user.js:3]: what's `App` here?

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  profileImage: DS.attr('string'),
+  profileLink: DS.attr('string'),
+  username: DS.attr('string'),
+  name: DS.attr('string'),
+  company: DS.attr('string'),
+  blogURL: DS.attr('string'),
+  followerCount: DS.attr('integer'),
```
Comment (twokul) [inline app/models/user.js:10]: please create a local `attr` variable

File: app/models/user.js (hunk 1 above)
Comment (randallkanna) [inline app/models/user.js:3]: @twokul Is this supposed to be the title of the app? IE: GitHubApp.User?

Comment (twokul) [inline app/models/user.js:3]: global variables are long gone. you should start using `ember generate` to generate models.

Comment (twokul) [inline app/models/user.js:3]: https://github.com/ember-cli/ember-cli/blob/master/blueprints/model/files/app/__path__/__name__.js#L3

Comment (randallkanna) [inline app/models/user.js:3]: Okay changed these. @twokul

File: app/models/user.js (hunk 2 above)
Comment (randallkanna) [inline app/models/user.js:10]: @twokul How do I create a local attribute variable?

**Pull request closed without merging**